src/api_proxy/service_control/request_info.h

```diff
@@ -145,9 +145,18 @@ enum ScResponseErrorType {
 struct ScResponseError {
   std::string name;
 
-  bool is_error_from_http_call;
+  bool is_network_error;
```

Comment:
I am just wondering if we can just put rpc_status here.   if rpc_status is not OK, it means nework failure.  otherwise it is error from server.

Reply:
This is not as straightforward and we don't need more information from rpc_status.